### ISSUE TITLE
I don't seem to see the code that calls the resource

### ISSUE THREAD

**Issue by xxdoc** [OlimilO1402/ResourcesTut#1]
Does this code just tell you how to generate resource files? I don't seem to see the code that calls the resource~~
^_^

**Comment by OlimilO1402** [comment]
Hi, this repo is one of my first repos on github, once a time I thought I had to do this tutorial, but it turned out to be something different ;-) what I originally wanted to show was, how to create resource files and how to use resource files.
in my first year on github nothing really happened and I thought, you know, there maybe are not many VB-devlopers left out there anymore. So I decieded to give up the work on it as it turned out to be very much work to do a tutorial about this subject. Maybe renaming it would help, don't quote me on that but It seems to me that it is not possible to rename a repo on github, do we really have to actually delete it and do a new repo, and the old name is spoiled for ever? So i decided to leave it as it is and come to it later if, in the rare case, somebody actually asks for it. ^_^

**Comment by xxdoc** [comment]
Indeed, very few people use vb6 now, and it is difficult to find information.
Thanks for your code. Maybe this is where the final belongs.
